packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -1736,6 +1744,28 @@ describe('DirectStripeRoutes', () => {
         isActive
       );
 
+    describe('handleCustomerCreatedEvent', () => {
```

Comment:
We should have a unit test for the condition that a customer is created without a userid in the metadata.

Reply:
Since we don't use the userid from the metadata, that unit test wouldn't be any different than the one below.